irtojs.ml

```diff
@@ -564,20 +564,20 @@ end
 module type WEB_COMPILER = sig
   val generate_program_page : ?cgi_env:(string * string) list ->
     (Var.var Env.String.t * Types.typing_environment) ->
-    Ir.program ->  string
+    Ir.program -> (string list) -> string
```

Comment:
I think a type alias for `string list` would be informative. For example 
```ocaml
type ext_deps = string list
```
or 
```ocaml
type foreign_stub = string
...
val generate_program_page : ?cgi_env:(string * string) list -> ... -> foreign_stub list -> string
```
or use an optional parameter whose default value is `[]`
```ocaml
val generate_program_page : ?cgi_env:(string * string) list -> ?foreign_stubs:string list -> ... -> string
```